src/processing/UndoRedo.java

```diff
@@ -214,8 +272,9 @@ private static void deleteFromStack(ArrayList <BoardObject> stack, ObjectId obje
 		   ListIterator <BoardObject> iterStack = stack.listIterator();
 		   while (iterStack.hasNext()) {
 			   BoardObject obj = iterStack.next();
-			   if (obj.getObjectId().equals(objectId))
+			   if (obj.getObjectId().equals(objectId)) {
 				   iterStack.remove();
+			   }
 		   }
 		   return;
```

Comment:
Fix indentation between this and next function.

Reply:
Fixed

Reply:
I cant see it updated. Can you check once?

Reply:
I didn't notice this and I fixed something else. Now, I have fixed it. Check it once.